### PR TITLE
gauging locks.

### PR DESCRIPTION
**Pull request by mihxil**

### CODE REVIEW

File: vpro-shared-monitoring/src/main/java/nl/vpro/monitoring/config/MonitoringConfig.java [hunk 1]
```diff
@@ -92,6 +95,15 @@ public PrometheusMeterRegistry globalMeterRegistry() {
                 new DiskSpaceMetrics(new File(folder)).bindTo(registry);
             }
         }
+        if (properties.isMeterLocks()) {
+            Gauge.builder("locks.currentCount", ObjectLockerAdmin.JMX_INSTANCE, ObjectLockerAdmin::getCurrentCount)
```
Comment (soarten) [inline vpro-shared-monitoring/src/main/java/nl/vpro/monitoring/config/MonitoringConfig.java:99]: Use lower case by convention zo current.count. Not sure wether "count" provides the strongest semantics here, does it mean: "lock active"?

File: vpro-shared-monitoring/src/main/java/nl/vpro/monitoring/config/MonitoringConfig.java [hunk 2]
```diff
@@ -92,6 +95,15 @@ public PrometheusMeterRegistry globalMeterRegistry() {
...
+                .description("The current number of locked objects")
+                .register(registry);
+            Gauge.builder("locks.maxConcurrency", ObjectLockerAdmin.JMX_INSTANCE, ObjectLockerAdmin::getMaxConcurrency)
```
Comment (soarten) [inline vpro-shared-monitoring/src/main/java/nl/vpro/monitoring/config/MonitoringConfig.java:102]: Same here lower case only. And what does max.concurrency mean?

Comment (mihxil) [reply vpro-shared-monitoring/src/main/java/nl/vpro/monitoring/config/MonitoringConfig.java]: I put the meaning in the description.

File: vpro-shared-monitoring/src/main/java/nl/vpro/monitoring/config/MonitoringConfig.java (hunk 1 above)
Comment (mihxil) [reply vpro-shared-monitoring/src/main/java/nl/vpro/monitoring/config/MonitoringConfig.java]: would perhaps be better, but I think its also worth somethng to follow the method names already in use.

**Pull request merged**